theme/theme.config

```diff
@@ -43,7 +43,7 @@
 @form       : 'default';
 @grid       : 'pxt';
 @menu       : 'pxt';
-@message    : 'default';
+@message    : 'pxt';
```

Comment:
Pull latest of master. Isn't this already in?

Reply:
It's because I did a cherry pick of the branch you merged that was fixing the color contrast on "V0" ;-)